### ISSUE TITLE
Server Crash when loading new chunks in Atum

### ISSUE THREAD

**Issue by Zerosk83** [NillerMedDild/Enigmatica6#3771]
### Modpack Version

0.5.20

### Describe your issue.

I'm trying to explore Atum on my dedicated server right now. I'm hosting it locally on the same PC I'm playing from.

I've been able to get into Atum without the server crashing, but as soon as I start trying to explore it crashes. I confirmed this after a bit more testing. If I jump in the portal and don't move, I can chat and use server commands etc, even save the game without issue. 
But as soon as I start exploring and load new chunks it hangs until the tick timeout response. 

I wanted to see if this was a potential character issue, so I tried using a pregen command with no one on the server and that crashes as well, immediately after loading the first batch of chunks
/pregen start gen radius ATUM CIRCLE 0 0 75 atum:atum

I see a few of the pregen message go by, but as soon as the first batch of chunks actually load things hang until the tick timout again

Here's 2 crash reports. 
[crash-2021-11-23_21.57.05-server.txt](https://github.com/NillerMedDild/Enigmatica6/files/7592625/crash-2021-11-23_21.57.05-server.txt)
[crash-2021-11-23_21.44.50-server.txt](https://github.com/NillerMedDild/Enigmatica6/files/7592626/crash-2021-11-23_21.44.50-server.txt)
.

### Crash Report

https://pastebin.com/JG9ZA0yU

### Have you modified the modpack?

Yes

### User Modifications

Added Bluemap. Tried removing it from the pack and pregenning again, no change, server still crashes

Changed server tick timeout to 90000 (90 seconds) from 60000 (60 seconds) in the server properties

### Did the issue happen in singleplayer or on a server?

Server

**Comment by NillerMedDild** [comment]
Thanks for the report. Please try setting server tick timeout to 0, hopefully the server will crash from the issue and we can see what it is.

**Comment by Zerosk83** [comment]
So I did as you requested and it doesn't officially crash. It just seems to hang indefinitely. It's been 20 minutes since things became unresponsive (can move in game for a while before being kicked, but can't interact and everything says "waiting for server" along the top. Eventually it kicks me with "Lost Connection"). I can't enter commands in the console, pressing enter just adds a new blank line. 

However, after about 10 minutes in this frozen state I tried to reconnect since the server showed as online still according to my client (it also shows me as online still 1/20). It attempts to join and I see some console messages showing that I'm attempting to connect, but it just fails after a minute or so. 

I'm not sure what else I can do to try and troubleshoot this or get you more info. Do you have anything else you'd like me to try?

**Comment by NillerMedDild** [comment]
Alright, damn. May I have the server's latest and debug logs please?

**Comment by Zerosk83** [comment]
So I moved my server folder from C:\Users\%USERNAME%\Desktop\Engimatica6 to C:\Servers\Enigmatica6
After the move, I bundled all the existing logs into a folder called "OLD".

I then started the server up again and attempted to recreate the issue so that I had fresh logs that didn't include as much user info. Once booted up I found that I wasn't able to recreate the issue. I've gone back and forth through the portal a few times and I can explore without issue now. 

I'm not really sure why it's working, but the issue seems to have disappeared for now.
